src/Workspaces/Core/Portable/CodeStyle/AnalyzerConfigOptionsExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal static class AnalyzerConfigOptionsExtensions
+    {
+        private readonly static MethodInfo _tryGetEditorConfigOptionMethodInfo = typeof(AnalyzerConfigOptionsExtensions).GetMethod("TryGetEditorConfigOption", BindingFlags.NonPublic | BindingFlags.Static);
+
+        public static bool TryGetOption<T>(this AnalyzerConfigOptions analyzerConfigOptions, Option<T> option, out T value)
```

Comment:
Can we rename these TryGetEditorConfigOption as well? That way it's clear this fetches just the editorconfig option?

Reply:
(the specific confusion being there's a lot of other calls to this and it's sometimes unclear what *those* are calling...)

Reply:
Doesn't look like this was applied?

Reply:
Good catch! =)